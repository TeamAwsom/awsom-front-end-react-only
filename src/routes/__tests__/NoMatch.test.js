import React from 'react';
import NoMatch from '../NoMatch';
import renderer from 'react-test-renderer';

describe('NoMatch', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<NoMatch />).toJSON();

    console.log(tree);
    expect(tree).toMatchSnapshot();
  });
});
