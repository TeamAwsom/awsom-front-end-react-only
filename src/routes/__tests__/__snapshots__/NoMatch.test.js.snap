// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoMatch renders as expected 1`] = `
<div
  className="container"
>
  <div
    className="match"
  >
    <h2>
      Sorry, we could't find a match for you.
    </h2>
    <p>
      Please send us an e-mail, and we'll connect you to a great teacher within 24 hours!
    </p>
    <div
      className="matchdetails"
    >
      <div>
        Email: HELLO@AWSOM.INFO
      </div>
      <div>
        Phone: (425) 951-9758
      </div>
    </div>
  </div>
</div>
`;
